### ISSUE TITLE
Hello, does your reimplementation have the close performance with the official one?

### ISSUE THREAD

**Issue by zoezhou1999** [fenghansen/KinD-pytorch#1]
Thank you for your code~

**Comment by fenghansen** [comment]
No, I suspect there is something wrong with original 'illumination smoothness' loss. I can't reimplemente it even though I use its official TensorFlow version.

**Comment by zoezhou1999** [comment]
Hi, I can reproduce decom net in official TensorFlow version. Train in cpu. tensorflow 1.14.0
